README.md

```diff
@@ -11,28 +12,50 @@ A React wrapper for the the [Handsontable](https://github.com/handsontable/hands
 7. [Other wrappers](#other-wrappers)
 
 ## Installation
+Depending on whether you're using [Handsontable](https://github.com/handsontable/handsontable) or [Handsontable Pro](https://github.com/handsontable/handsontable-pro), you'll need to install:
 
-For detailed installation instructions, please take a look at our wiki under ["Installation guide"](https://github.com/handsontable/react-handsontable/wiki/Installation-guide).
+```sh
+npm install @handsontable/react handsontable
+```
 
-## Building
-If you used npm to download the library, you should be good to go, but if you want to make a build yourself, go to the directory where `react-handsontable` source is located and run:
+or
 
 ```sh
-npm run build
+npm install @handsontable-pro/react handsontable-pro
 ```
 
-This will create a `/dist/` directory with `react-handsontable.js` and `react-handsontable.min.js` for you to use. 
+Installing the `react-handsontable` package will have the same effect as installing `@handsontable/react`, which is the wrapper for the CE version.
+
+**Important:** The wrapper does NOT contain `Handsontable`/`Handsontable Pro`, that's why it needs to be installed separately.
+
+For more detailed installation instructions, please take a look at our wiki under ["Installation guide"](https://github.com/handsontable/react-handsontable/wiki/Installation-guide).
```

Comment:
If we link to Wiki it will stay with us for a long time or older NPM releases will have broken links. Can't we link to our documentation already?